fieldservice/views/fsm_order.xml

```diff
@@ -0,0 +1,491 @@
+        <field name="type">calendar</field>
+        <field name="arch" type="xml">
+            <calendar
+                string="FSM Orders"
+                date_start="scheduled_date_start"
+                date_delay="scheduled_duration"
+                color="stage_id"
+            >
+                <field name="name" />
+                <field name="stage_id" />
+                <field name="location_id" />
+                <field name="phone" />
+                <field
+                    name="person_id"
+                    write_model="fsm.person.calendar.filter"
+                    write_field="fsm_person_id"
+                    avatar_field="image_small"
+                />
+                <field name="person_phone" />
+                <field name="city" />
+                <field name="state_name" />
+            </calendar>
+        </field>
+    </record>
+    <record id="action_fsm_dash_order" model="ir.actions.act_window">
+        <field name="name">Orders</field>
+        <field name="res_model">fsm.order</field>
+        <field name="view_mode">kanban,tree,form,calendar</field>
+        <field name="search_view_id" ref="fsm_order_search_view" />
+        <field name="help" type="html">
+            <p class="o_view_nocontent_smiling_face">
+                Create an Order.
+            </p>
+        </field>
+    </record>
+    <!-- Reporting -->
+    <record id="fsm_order_graph_view" model="ir.ui.view">
+        <field name="name">fsm.order.graph</field>
+        <field name="model">fsm.order</field>
+        <field name="arch" type="xml">
+            <graph string="Service Orders" type="bar">
+                <field name="stage_id" type="row" />
+            </graph>
+        </field>
+    </record>
+    <record id="fsm_order_pivot_view" model="ir.ui.view">
+        <field name="name">fsm.order.pivot</field>
+        <field name="model">fsm.order</field>
+        <field name="arch" type="xml">
+            <pivot string="Service Orders" display_quantity="true">
+                <field name="stage_id" type="row" />
+            </pivot>
+        </field>
+    </record>
+    <record id="action_fsm_report_order" model="ir.actions.act_window">
+        <field name="name">Orders</field>
+        <field name="res_model">fsm.order</field>
+        <field name="view_mode">graph,pivot</field>
+        <field name="help" type="html">
+            <p>
+                Orders Reports.
+            </p>
+        </field>
+    </record>
+    <record id="action_report_fsm_order" model="ir.actions.report">
+        <field name="name">Service Order</field>
+        <field name="model">fsm.order</field>
+        <field name="report_type">qweb-pdf</field>
+        <field name="report_name">fieldservice.report_fsm_order</field>
+        <field name="report_file">fieldservice.report_fsm_order</field>
+        <field name="print_report_name">'Service Order - %s' % (object.name)</field>
+        <field name="binding_type">report</field>
```

Comment:
```suggestion
        <field name="binding_model_id" ref="model_fsm_order" />
```
In order to allow the report to be visible under Print menu for FSM order
[https://www.odoo.com/documentation/14.0/reference/actions.html#report-actions-ir-actions-report](https://www.odoo.com/documentation/14.0/reference/actions.html#report-actions-ir-actions-report)